### ISSUE TITLE
Flickering <Transition>  in release mode

### ISSUE THREAD

**Issue by fdiakh** [leptos-rs/leptos#960]
Transitions don't seem to work properly in release mode. The child goes blank while the resource is reloading and comments are accumulating in the HTML each time it is re-rendered. In debug mode the problem does not appear.  The child remains displayed while the resource is reloading, there is no flickering or accumulating comments.

This can be reproduced as follows:

```rust
use leptos::*;

fn update_counter_bg(mut value: i32, step: i32, sig: WriteSignal<i32>) {
    sig.set(value);
    value += step;
    leptos::set_timeout(
        move || {
            update_counter_bg(value, step, sig);
        },
        std::time::Duration::from_millis(2000),
    );
}

async fn get_resource(value: i32) -> i32 {
    gloo_timers::future::TimeoutFuture::new(1000).await;
    value
}


#[component]
pub fn SimpleCounter(
    cx: Scope,
    initial_value: i32,
    step: i32,
) -> impl IntoView {
    let (value, set_value) = create_signal(cx, initial_value);

    // update the value signal periodically
    update_counter_bg(initial_value, step, set_value);

    // simulate a resource that is fetched from the server when the signal is triggered
    let resource = create_resource(cx, value, get_resource);

    view! { cx,
        <Transition fallback=move || {
            view! { cx, <p>"Loading..."</p> }
        }>
            <div>
                <h3>
                    {move || {
                        resource
                            .with(
                                cx,
                                |value| {
                                    view! { cx, {format!("{value}")} }
                                },
                            )
                    }}
                </h3>
            </div>
        </Transition>
    }
}
```

**Comment by gbj** [comment]
Just for the sake of clarity, are you talking about client-side-rendering/CSR mode or hydrate mode, or both?

**Comment by gbj** [comment]
Thanks. I think this is fixed by #967 but I'll check back in when that's done.

**Comment by gbj** [comment]
This appears to have broken against at some point, I'll have to take another look.
